public/app/app.ts

```diff
@@ -179,8 +182,9 @@ function initEchoSrv() {
         );
       }
 
-      const loadMetric = performance.getEntriesByName(loadMetricName)[0];
-      reportPerformance(loadMetric.name, Math.round(loadMetric.startTime + loadMetric.duration) / 1000);
+      reportMetricPerformance(loadMetricName);
```

Comment:
What is this actually measuring? We mark it on line 174 and then report it here. It seems like it is only measuring how long it takes to report the paint metrics?

Reply:
It measures the time up until that line is executed - from initial page load to window load event.